docs/ADRs/012-tad-tool.md

```diff
@@ -0,0 +1,56 @@
+# ADR-0012 Tool for Transparency of Algorithmic Decision making
```

Comment:
We keep project ADRs in Projects (which, in this case, happens to be in the same repo, on a different path).

Not sure we need an ADR to decided on that?

Customary, every new series of ADRs starts with "an ADR on the fact that we use ADRs".

Reply:
"We keep project ADRs in Projects" sounds like a decision/WoW. An ADR is a bit much, but maybe in Way-of-Working we can keep a list of such decisions?

Also fine with using an ADR to declare the ADR, point is, the file is duplicated instead of f.e. referenced which could (even if unlikely) lead to different files.. so I wondered if referencing it may be a solution to that issue?

Reply:
If you think it warrants and ADR or note somewhere: we should to that. 

For the second: I don't think it matters tbh.